### PR TITLE
Sauce labs iOS automation

### PR DESCRIPTION
**Pull request by Piyush23786**

### CODE REVIEW

File: SplunkRumWorkspace/SplunkRum/SplunkRum/RetryExporter.swift [hunk 1]
```diff
@@ -63,10 +63,12 @@ class RetryExporter: SpanExporter {
 
     func flush() -> SpanExporterResultCode {
         _ = attemptPendingExport()
+       // exportedSpans.removeAll()
```
Comment (seemk) [inline SplunkRumWorkspace/SplunkRum/SplunkRum/RetryExporter.swift:66]: Should this be commented out?

File: SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift [hunk 2]
```diff
@@ -178,7 +178,8 @@ let globalAttributesLock = NSLock()
 
 let splunkLibraryLoadTime = Date()
 var splunkRumInitializeCalledTime = Date()
-
+// swiftlint:disable missing_docs
+public var sessiontoken: String = ""
```
Comment (seemk) [inline SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift:182]: I don't think the changes in this file are used anywhere

Comment (Piyush23786) [reply SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift:182]: removed

File: SplunkRumWorkspace/SplunkRum/SplunkRum/RetryExporter.swift (hunk 1 above)
Comment (Piyush23786) [reply SplunkRumWorkspace/SplunkRum/SplunkRum/RetryExporter.swift:66]: It's added for testing.
Removed

**Pull request closed without merging**